cognite/powerops/case/shop_run_event.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import annotations
+
+import json
+from dataclasses import dataclass
+from functools import cached_property
+from typing import ClassVar, Dict, Optional
+
+from cognite.client.data_classes import Event
+
+from cognite.powerops.utils.common import unique_str
+
+
+@dataclass
+class ShopRunEvent:
+    """
+    A cut-down variant of `common.workflow_utils.ShopRun` from
+    power-ops-functions repo. This variant has no workflow event and
+    no mappings.
+    """
+
+    event_type: ClassVar[str] = "POWEROPS_PROCESS_REQUESTED"
+    process_type: ClassVar[str] = "POWEROPS_SHOP_RUN"
+    watercourse: str
+    starttime: str
+    endtime: str
+    timeresolution: Optional[Dict[str, int]] = None
```

Comment:
Since you import `from __future__ import annotations`

```suggestion
    timeresolution: Optional[dict[str, int]] = None
```

Reply:
Sure, old habits.
BTW, `from __future__ import annotations` is only for delayed annotation evaluation, I don't think it affects `dict` vs `Dict`.